# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxPlatformGtk.h
# Commit: 4f5567292c37
# Full Hash: 4f5567292c37eedabac14b615f41c961cfc244f3
# Author: stransky <stransky@redhat.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1751987
# File Overlap Count: 1
# Description:
#   Bug 1751987 [Linux] Add VA-API feature r=aosmond
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D139301
# ==============================================================================

diff -r b7ea622c1205 -r 4f5567292c37 gfx/thebes/gfxPlatformGtk.h
--- a/gfx/thebes/gfxPlatformGtk.h	Wed Feb 23 11:47:06 2022 +0000
+++ b/gfx/thebes/gfxPlatformGtk.h	Wed Feb 23 11:47:07 2022 +0000
@@ -68,6 +68,7 @@
  protected:
   void InitX11EGLConfig();
   void InitDmabufConfig();
+  void InitVAAPIConfig();
   void InitPlatformGPUProcessPrefs() override;
   void InitWebRenderConfig() override;
   bool CheckVariationFontSupport() override;